lib/UhstSocket.ts

```diff
@@ -49,7 +49,12 @@ export class UhstSocket {
         this._ee.off(eventName, handler);
     }
 
-    send = (message: string) => {
```

Comment:
I am not sure why you declare class methods like this, but for method overloading, it is better to define class methods with the normal syntax.

Reply:
it's because of "this" binding, if I declare it with normal syntax then I found that I have to do .bind(this) or otherwise `this` will be undefined in some cases. Thanks for your contribution, I will test and if no issues with this binding will accept.

Reply:
This is the issue I am referring to: https://yiou.me/blog/posts/type-script-this

Reply:
One fix would be to add this to the constructor:

```typescript
this.send = this.send.bind(this);
```

Reply:
I can add this to the pr if you want. This would make sure `this` is always defined on the send function.

Reply:
I create a separate issue for it but up to you if you want to add in the same PR.